requirements.txt

```diff
@@ -0,0 +1,2 @@
+# Drakkar-Software requirements
+OctoBot==0.4.27
```

Comment:
Should we add jinja or we should install it through OctoBot requirements ?

Reply:
good question, here is jinja and aiohttp that we are using as well, i'll  add them